catalog/src/main/kotlin/com/adevinta/spark/catalog/CatalogApp.kt

```diff
@@ -207,8 +208,17 @@ internal fun ComponentActivity.CatalogApp(
             ) {
                 val coroutineScope = rememberCoroutineScope()
                 val homeScreenValues = CatalogHomeScreen.entries
+                val initialPage = LocalActivity.current?.intent?.data?.let {
```

Comment:
IMO `initialPage` should be `CatalogHomeScreen`, and the call to `.ordinal` should be in the `rememberPagerState` call only.

Reply:
Done in 7b20af4f02332c94d5dab6704f845e144ae35eb8